packages/mask/src/plugins/Trader/SNSAdaptor/trending/TrendingViewDeck.tsx

```diff
@@ -148,6 +148,7 @@ export function TrendingViewDeck(props: TrendingViewDeckProps) {
     }, [account, trending?.coin?.symbol])
     // #endregion
 
+    const isNFT = dataProvider === DataProvider.NFTSCAN
```

Comment:
Why does it decide on the token type by comparing with the data provider?

Reply:
Now use `coin.type` instead.